en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -563,50 +548,52 @@ Based on the partisan indicators assigned to the data, blue plot points represen
 
 The small cluster of blue out to the far right of this visualization suggests that some of the most polarizing comments were added on videos from left-leaning channels. Otherwise, based on this visualization, the channel's political affiliation does not seem to be a strong predictor of the commenters' political positions. 
 
 ## Conclusion
 
-In this lesson, you have learned how to download a large corpus of YouTube video comments, process the text data, analyze the comments using the Wordfish algorithm, and produce several insightful visualizations. You can reuse the code in this lesson on your own YouTube comment dataset - to download the code for easy re-use, see the attached [R script](/ph-submissions/assets/text-mining-youtube-comments/youtube.R).
+In this lesson, you learned how to download a large corpus of YouTube video comments, process the text data, analyze the comments using the Wordfish algorithm, and produce several insightful visualizations. You can reuse the code in this lesson on your own YouTube comment dataset - to download the code for easy re-use, see the attached [R script](/ph-submissions/assets/text-mining-youtube-comments/youtube.R).
 
-If you used this lesson's data, you can tell from these three visualizations that a broadly similar set of topics is discussed on comment threads for both left-leaning and right-leaning YouTube videos focused on police brutality and police funding in the United States since 2020. You have also seen that you can interpret these visualizations to discover which words determined the scale created by the Wordfish model, and which videos contributed to each pole of that scale. 
+If you used this lesson's dataset, you were able to tell from these visualizations that a broadly similar set of topics is discussed on comment threads for both left-leaning and right-leaning YouTube videos focused on police brutality and police funding in the United States since 2020. You also saw that you can interpret these visualizations to discover which words determined the scale created by the Wordfish model, and which videos contributed to each pole of that scale. 
 
 Further analysis of these visualizations, and more granular analyses of the underlying Wordfish model, will enable more complex interpretations of the corpus' textual meaning. The fact that Wordfish can be useful for understanding the strange type of discourse that appears in YouTube comments is a fascinating revelation of its own.
 
 ## Endnotes
 
-[^1]: Here is a selected bibliography:
+[^1]: Rieder, Bernhard. _YouTube Data Tools_. Version 1.42. Platform Digitale Infrastructuur SSH. <https://ytdt.digitalmethods.net>.
+
+[^2]: Here is a selected bibliography on this topic:

```

Comment:
I agree with you that the endnotes are currently a little unorthodox, as 2, 3 and 4 are formatted as 'selected bibliographies' on a certain topic. 

I believe this makes sense in the context of the paragraph they are found in, but we could do it differently if you preferred. 

Do you have any suggestions?

Was there a particular list you wanted to add <https://www.amazon.com/YouTube-Participatory-Culture-Digital-Society/dp/0745660193> to? Or can we point to it using an endnote, from within the lesson body?

Reply:
It could be an endnote from within the lesson body linked relatively early when we mention scholarship on Youtube, since it's the closest thing to a 'textbook'. I'll double check the bibliographies but as long as the link brings people to the right place to find them, we're fine with whatever works.